spec/Section 3 -- Type System.md

```diff
@@ -1692,6 +1686,20 @@ While defining a directive, it must not reference itself directly or indirectly:
 directive @invalidExample(arg: String @invalidExample) on ARGUMENT_DEFINITION
 ```
 
+**Built-in Directives**
+
+GraphQL implementations should provide the `@skip` and `@include` directives.
+When returning the set of directives from the `__Schema` introspection type, both
+`@skip` and `@include` directives must be included.
```

Comment:
Since `@skip` and `@include` directives only *should* be provided, a service can omit them, if for some reason its executor cannot handle them.

They should be present in introspection if the service supports them.